src/main/java/io/anserini/index/generator/GeneratorExpection.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Anserini: A Lucene toolkit for replicable information retrieval research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.index.generator;
+
+/**
+ * The root class of all exceptions that indicate abnormal behavior by {@link LuceneDocumentGenerator}.
+ */
+public class GeneratorExpection extends Exception {
```

Comment:
GeneratorException*? (Here and everywhere else we have "GeneratorExpection")

Reply:
oops, will fix.